tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1945,6 +1946,13 @@ def random_hue(image, max_delta, seed=None):
 
   `max_delta` must be in the interval `[0, 0.5]`.
 
+  Usage Example:
+  ```python
+  >> import tensorflow as tf
```

Comment:
No need for import tensorflow as tf

Seed the random operations and use small images so it becomes a valid doctest.